content/docs/components/stat/theming.mdx

```diff
@@ -3,4 +3,104 @@ id: stat
 scope: theming
 ---
 
-TODO
+The Stat component is a multipart component. The styling needs to be applied to each part specifically.
+
+> To learn more about styling multipart components, visit the
+> [Component Style](/docs/styled-system/component-style#styling-multipart-components)
+> page.
+
+This component has five parts: `container`, `label`, `helpText`, `icon`,  and `number`. You can find more information in the source for the styles [here](https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/theme/src/components/stat.ts).
```

Comment:
I've prepared an anatomy illustration for you. You can grab the code [from this gist](https://gist.github.com/nikolovlazar/ee0d3372065ca045a24bb8d1694e0520).

Also, make a new section for the anatomy:
```markdown
## Anatomy

- A: `container`
- B: `label`
- C: `helpText`
- D: `icon`
- E: `number`

[PLACE THE ILLUSTRATION CODE HERE]
```

Reply:
Thanks, i will include it